### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**
#### Summary

When regenerating the dictionary for #17631 I noticed a few new ones that made sense to split off as a separate PR

### CODE REVIEW

File: .github/workflows/cspell-problem-matcher.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "cspell",
+      "pattern": [
+        {
+          "regexp": "^(.*):(\\d+):(\\d+)\\s+\\-\\s+(.*)$",
```
Comment (bsmth) [inline .github/workflows/cspell-problem-matcher.json:7]: What does this do?

File: project-words.txt [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+appinstalled
+attributionsourceid
+autocrlf
+availabilitychanged
+backgroundfetchabort
+backgroundfetchclick
+backgroundfetchfail
+backgroundfetchsuccess
+backported
+Backported
```
Comment (bsmth) [inline project-words.txt:15]: Is it necessary to add upper+lower variants to the project dict?

File: .github/workflows/cspell-problem-matcher.json (hunk 1 above)
Comment (nschonni) [reply .github/workflows/cspell-problem-matcher.json:7]: Sorry, this shouldn't have been included, it should be part of the other PR in the descritpion

**Comment by nschonni** [comment]
@bsmth I've rebased this one down to just the fixes. The cSpell setup remains in the other PR

**Review by bsmth — APPROVED** [review]
Looks good, thanks @nschonni 👍🏻

**Pull request merged**